### DISCUSSION TITLE
Sonoff ZigBee 3.0 USB Dongle Plus v2 (ZBDongle-E) Come Router

### DISCUSSION THREAD

**Discussion by diaz66** [indomus/forum#2188] [ZigBee, Z-Wave, Thread, Matter]
Grazie come al solito per le vostre guide, fondamentali in tanti casi.

Essendo alle prime armi con ZegBee e a proposito di questa chiavetta, (ne possiedo già una ed ho appena iniziato con ZegBee2Mqtt) mi sembra di aver capito che, con firmware giusto, può funzionare anche come router (ripetitore di segnale). 

Perchè ho notato che se posiziono il sensore di temperatura SNZB-02 abbastanza lontano dalla chiavetta , questo si disconnette e a volte scompare pure dalla lista dispositivi su ZegBee2Mqtt e non ricompare più, anche se lo avvicino alla chiavetta.

Ho già fatto un pò di prove con le frequenze ma cambia poco (attualmente uso la 26).

Quindi penso che dovrò inserire un router.

Quindi questa chiavetta con firmware router poi dovrò solo alimentarla tramite USB e posizionarla il meglio possibile ?

Oppure esistono altri end device da utilizzare allo scopo e che qualcuno ha provato che funzionano bene ?

Grazie per l' attenzione.
Un Saluto a tutti.

**Comment by snakuzzo** [comment]
Innanzi tutto si scrive "Zigbee" e non "Zegbee"

> questo si disconnette e a volte scompare pure dalla lista dispositivi su ZegBee2Mqtt e non ricompare più

Guarda...questo non capita (dovrebbe capitare mai). Anche se lo tieni lontano da giorni, poi non dovrebbe "sparire" dalla lista dei device di zigbee2mqtt. Nel caso specifico quanto scritto [qui](https://www.zigbee2mqtt.io/devices/SNZB-02.html#troubleshoot) potrebbe essere proprio legato ad un "problema noto" del device.
Poi devi considerare che i device a batteria non è che li avvicini e "di botto" riprendono a funzionare. I device a batteria vanno in deep sleep e si riattivano solo quando devono riattivarsi...nel tuo caso lo faranno quando rileveranno un cambio di temperatura superiore ad una determinata differenza (non so quale) rispetto alla temperatura precedentemente rilevata.

> Quindi questa chiavetta con firmware router poi dovrò solo alimentarla tramite USB e posizionarla il meglio possibile ?
Oppure esistono altri end device da utilizzare allo scopo e che qualcuno ha provato che funzionano bene ?

Sì, puoi usare un'altra chiavetta simile con firmware router, ma considera che in linea di massima ti basta anche un qualsiasi device zigbee alimentato NON a batteria...quelli solitamente sono tutti router. Sicuramente con chiavette/device con chipset Texas Instruments e flashati con fw router ti permettono di evitare problemi di "non compatibilità" tra device di marchi diversi.

**Reply by diaz66** [reply]
...e vero, ahaha nel titolo l'ho scritta bene e poi...Grazie tanto per la risposta dettagliatissima.
Comunque si, ci sto giocando e ti confermo che mi è proprio scomparso dalla lista dei dispositivi. Me la fatto 1 volta sicuro, forse 2...non ricordo bene....
Proverò ad aspettare più tempo dopo averlo riavvicinato alla chiavetta, a meno che non mi scompare dalla lista anche se, mi pare che dopo circa 2 ore , non lo vedevo ancora.
